users/vieting/experiments/switchboard/hybrid/feat/specaug_sorted.py

```diff
@@ -0,0 +1,153 @@
+
+def sort_filters_by_center_freq(x, return_sorted_filters=True):
+    """
+    This function either sorts the filters by their center frequency and returns them,
+    or it returns the indices that would sort the filters.
+    :param tf.Tensor x: The filter layer to sort.
+    :param bool return_sorted_filters: Whether to return the sorted filters or the sorted indices.
+    :return: Sorted filters or sorted indices.
+    """
+    x = tf.convert_to_tensor(x)  # (256, 1, 150) = (N, 1, C)
```

Comment:
```suggestion
    x = tf.convert_to_tensor(x)  # (N, 1, C)
```
Hard-coded numbers are not needed. It would be nice though to shortly mention what N and C stand for (N is filter size and C the number of channels iirc).